app/src/main/java/com/orange/ods/app/ui/components/buttons/ButtonsText.kt

```diff
@@ -71,36 +78,56 @@ fun ButtonsText(customizationState: ButtonCustomizationState) {
                 )
             }
 
-            CodeImplementationColumn(modifier = Modifier.padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.screen_horizontal_margin))) {
-                FunctionCallCode(
-                    name = OdsComposable.OdsTextButton.name,
-                    exhaustiveParameters = false,
-                    parameters = {
-                        enum("style", textButtonStyle.value)
-                        if (hasFullScreenWidth) fillMaxWidth()
-                        if (hasLeadingIcon) {
-                            classInstance<OdsButton.Icon>("icon") {
-                                painter()
+            CodeImplementationColumn(
+                modifier = Modifier.padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.screen_horizontal_margin)),
+                composeContent = {
+                    FunctionCallCode(
+                        name = OdsComposable.OdsTextButton.name,
+                        exhaustiveParameters = false,
+                        parameters = {
+                            text(buttonText)
+                            enum("style", textButtonStyle.value)
+                            if (hasFullScreenWidth) fillMaxWidth()
+                            if (hasLeadingIcon) {
+                                classInstance<OdsButton.Icon>("icon") {
+                                    painter()
+                                }
                             }
+                            if (!isEnabled) enabled(false)
                         }
-                        if (!isEnabled) enabled(false)
+                    )
+                },
+                xmlContent = {
+                    CodeBackgroundColumn {
+                        XmlViewTag(
+                            clazz = com.orange.ods.xml.component.button.OdsTextButton::class.java,
+                            xmlAttributes = {
+                                id("ods_text_button")
```

Comment:
There is a typo in the `ods_text_button.xml` file where the ID is `ods_textbutton`, but the correct one is `ods_text_button` as you wrote it.